db/migrations/20210115171617-create-usuario.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('Usuarios', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      nombre: {
+        allowNull: false,
+        type: Sequelize.STRING,
+      },
+      apellido: {
+        allowNull: false,
+        type: Sequelize.STRING,
+      },
+      contrasenia: {
+        allowNull: false,
+        type: Sequelize.STRING,
+      },
+      dni: {
+        allowNull: false,
+        type: Sequelize.INTEGER,
+        unique: true,
+      },
+      telefono: {
+        allowNull: false,
+        type: Sequelize.INTEGER,
+      },
+      email: {
+        allowNull: false,
+        type: Sequelize.STRING,
+        unique: true,
+      },
+      rol: {
+        allowNull: false,
+        type: Sequelize.STRING,
```

Comment:
Perdón, esta se me pasó en la review anterior, pero debería ser un ENUM tanto acá como en el modelo:

```suggestion
        type: Sequelize.ENUM('invitado', 'estudiante', ...)
```